cypress/dependencies.bzl

```diff
@@ -1,37 +0,0 @@
-"""Starlark helper to fetch rules_cypress dependencies.
-
-Should be replaced by bzlmod for users of Bazel 6.0 and above.
-"""
-
-load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
-load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-
-def http_archive(name, **kwargs):
-    maybe(_http_archive, name = name, **kwargs)
-
-def rules_cypress_dependencies():
```

Comment:
Does this mean we're dropping non-bzlmod support?

Reply:
That was my plan, but I'll defer the decision an offline discussion